### PR TITLE
Docs/wat function comments

### PR DESCRIPTION
**Pull request by bushidocodes**
- Adds Note about how to find documentation for logging to console. This is appended to instructions in the web UI, but hidden for CLI users.
- Add Doxygen-style function comments in order to help disambiguate the module API/ABI called by the test runner.

### CODE REVIEW

File: exercises/practice/acronym/acronym.wat [hunk 1]
```diff
@@ -1,6 +1,20 @@
+;;
+;; Note to CLI Users: Read 'exercises/shared/.docs/debug.md' for instructions
```
Comment (ErikSchierboom) [inline exercises/practice/acronym/acronym.wat:2]: Students won't have access to this file without checking out the wasm repo, so I think we should try a different approach.
What I think we should do is to duplicate the contents from `exercises/shared/.docs/debug.md` into `exercises/shared/.docs/help.md`, which is a CLI-only file that ends up getting downloaded to a `HELP.md` document in the downloaded exercise's directory. See https://exercism.org/docs/building/tracks/shared-files#h-file-help-md The duplication isn't great, but if that bothers you too much, we could also create a new document in the `docs` folder and link to that document. But having this inlined is probably better.

Comment (bushidocodes) [reply exercises/practice/acronym/acronym.wat]: Sounds like a good approach to me! I removed the prepended block comments in source and appended the contents of `debug.md` to `help.md`

File: exercises/shared/.docs/help.md [hunk 2]
```diff
@@ -4,3 +4,274 @@ To get help if you're having trouble, you can use one of the following resources
 
 - [/r/WebAssembly](https://www.reddit.com/r/WebAssembly/) is the WebAssembly subreddit.
 - [Github issue tracker](https://github.com/exercism/wasm/issues) is where we track our development and maintenance of Javascript exercises in exercism. But if none of the above links help you, feel free to post an issue here.
+
+# How to Debug
```
Comment (ErikSchierboom) [inline exercises/shared/.docs/help.md:8]: The headings should increase one level:

```suggestion
## How to Debug
```

This also applies to the other headings.

**Review by ErikSchierboom — APPROVED** [review]
Great!

**Pull request merged**